packaging/deb/changelog

```diff
@@ -0,0 +1,5 @@
+poplog (0.2-1) UNRELEASED; urgency=low
```

Comment:
No idea what we're going to do here when we tag new releases. See whether debian requires a new entry for each version